libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/ComponentLabelsContext.tsx

```diff
@@ -0,0 +1,28 @@
+// (C) 2021 GoodData Corporation
+
+import React, { useContext } from "react";
+import { IShareDialogLabels } from "../types";
+import { IComponentLabelsProviderProps } from "./types";
+
+const labels: IShareDialogLabels = {
```

Comment:
This should be better named as `defaultLabels`. I'm not sure if some better values should not be used for example "access-type-label", etc.

Reply:
done